packages/next/tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "moduleResolution": "node",
     "jsx": "react-jsx",
     "stripInternal": true,
-    "verbatimModuleSyntax": true
+    "verbatimModuleSyntax": true,
+    "skipLibCheck": true
```

Comment:
I think the `skipLibCheck` additions are leftovers from when you had [updated a few eslint dependencies](https://github.com/vercel/next.js/commit/2ded7131308fc6bc92027265ed933cb352d1d129) which led to duplicate and conflicting installs of `@types/eslint`. Should be revertable, I guess?